src/docfx/build/metadata/MetadataProvider.cs

```diff
@@ -87,6 +87,18 @@ internal class MetadataProvider
             }
         }
 
+        public List<Error> Validate()
+        {
+            var errors = new List<Error>();
+            foreach (var validator in _schemaValidators)
+            {
+                errors.AddRange(validator.PostValidate());
+            }
+
+            // todo: exclude files with errors from output
```

Comment:
With #5621 merged, can remove this TODO:

Reply:
great!